src/Stream.php

```diff
@@ -60,7 +60,7 @@ public function __construct($stream, $mode = 'r')
     public function __toString()
     {
         if (! $this->isReadable()) {
-            return '';
+            throw new RuntimeException('Stream is not readable');
```

Comment:
Actually, we cannot throw exceptions here; see [the PHP manual](http://php.net/manual/en/language.oop5.magic.php#object.tostring). The correct behavior here is to return an empty string.